malaba6/E-Rental
# #167398979 built user sign in endpoint

#### What does this PR do?
It creates a user sign-in endpoint.

#### Description of Task to be done?
1. It adds a user sign-in controller in the auth controller file.
2. It validates the information submitted during sign-in such as email and password.
3. It adds a route for user sign-in.

#### How should this be manually tested?
1. Start the **Server** and **POSTMAN**.
2. **Browse** the following endpoint route: **localhost:3000/api/auth/signin**.
3. Make sure that you have an account or create one in advance.
4. Submit the required information - **email** and **password** - with **POST** method. 
5. Check if the validations work once you don't submit any of the required information.
6. After passing the validations, you should receive a **200 HTTP Status Code** along with the information related to your account such as firstname, lastname, and email.
